single_cell_condensed_sdrf.sh

```diff
@@ -130,31 +193,39 @@ if [ "$replicates" = true ]; then
 fi
 
 zoomaOption="-z"
-if [ -n "$SKIP_ZOOMA" ]; then
+if [ -n "$skipZooma" ]; then
   zoomaOption=""
 fi
 
-condense_sdrf.pl -e $EXP_ID $technicalReplicatesOption -sc $zoomaOption -o $ATLAS_SC_EXPERIMENTS/$EXP_ID/
-export CONDENSED_SDRF_TSV=$ATLAS_SC_EXPERIMENTS/$EXP_ID/$EXP_ID.condensed-sdrf.tsv
+# Pass the same exclusions file to both downstream scrips
+exclusions=
+
+if [ -e "$zoomaExclusions" ]; then
+exclusions=" -x $zoomaExclusions"
+fi
+condense_sdrf.pl -e $expId -fi $idfFile $technicalReplicatesOption -sc $zoomaOption -o $outputDir $exclusions
+
+export CONDENSED_SDRF_TSV=$outputDir/$expId.condensed-sdrf.tsv
 # Explode condensed SDRF for droplet experiments from RUN_ID to RUN_ID-CELL_ID.
-if [ -f $ATLAS_SC_EXPERIMENTS/$EXP_ID/cell_to_library.txt ]; then
+if [ -f $cellToLib ]; then
   use_run_id_cell_id_In_condensed
 fi
 
 # Explode condensed SDRF with inferred cell type
-export CELLTYPES=$ATLAS_PROD/singlecell/experiment/$EXP_ID.cells.txt
+export CELLTYPES=$ATLAS_PROD/singlecell/experiment/$expId.cells.txt
```

Comment:
Probably this needs to follow the same spirit of resolving the file based on where the IDF is.

Reply:
Done!